packages/karma-runner/src/karma-plugins/stryker-reporter.ts

```diff
@@ -56,6 +58,11 @@ export class StrykerReporter implements karma.Reporter {
     this.runTask?.resolve(this.collectRunResult());
   };
 
+  public configureHitLimit(hitLimit: number | undefined): void {
+    this.hitLimit = hitLimit;
+    this.hitCount = undefined;
```

Comment:
Wouldn't it be more logical if the `hitCount` defaults to 0? Or would you prefer `undefined` because it simply may not have been set?

Reply:
Yeah, I guess `0` would work, but want to keep it `undefined`. I think that makes more sense.